### ISSUE TITLE
[question] what is the final mAP on VOC dataset?

### ISSUE THREAD

**Issue by vuhonganh** [yhenon/keras-frcnn#155]
Hi, can someone provide how many mAP this implementation can achieve when well trained? I trained for a while (around 100k iterations on GTX 1080Ti) and only reached 0.4 mAP which is not good for VOC dataset.

**Comment by yhenon** [comment]
I am still sorting out some issues in the codebase - I got about 0.61 map on pascal voc after training for 250k iterations, and annealing the learning rate carefully. That's still less than expected (should be ~0.73 map).

**Comment by olaff09** [comment]
Hi @yhenon,

Could you tell me what kind of changes are you making to increase the mAP?

**Comment by vuhonganh** [comment]
@yhenon I think there will be a lower performance when you use sklearn to compute AP. Here is the [discussion](https://github.com/scikit-learn/scikit-learn/issues/4577) and the sklearn maintainers decide **not to implement** the interpolated version (which is used in Pascal VOC)

**Comment by vuhonganh** [comment]
Hi @yhenon, based on this [implementation](https://github.com/endernewton/tf-faster-rcnn/blob/master/lib/datasets/voc_eval.py) then your implementation using sklearn is different. Could you provide from which reference (if any) you use to implement your get_map(pred, gt, f) in measure_map.py?
